lib/iris/tests/test_analysis_calculus.py

```diff
@@ -733,19 +760,22 @@ def test_contrived_spherical_curl2(self):
         np.testing.assert_array_almost_equal(
```

Comment:
I've been tending to convert these calls to _shared_utils for sake of consistency.

Reply:
bcaca28a6a14cf62f30e034db5b6aa02e1029303